src/test/java/com/coravy/hudson/plugins/github/GithubLinkActionFactoryTest.java

```diff
@@ -0,0 +1,51 @@
+package com.coravy.hudson.plugins.github;
+
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.empty;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import com.coravy.hudson.plugins.github.GithubLinkAction.GithubLinkActionFactory;
+
+import hudson.model.Action;
+
+public class GithubLinkActionFactoryTest {
+    @Rule
+    public JenkinsRule rule = new JenkinsRule();
+
+    private GithubLinkActionFactory factory = new GithubLinkActionFactory();
+
+    private WorkflowJob createExampleJob() throws IOException {
+        return rule.getInstance().createProject(WorkflowJob.class, "example");
+    }
+
+    @Test
+    public void shouldCreateGithubLinkActionForJobWithGithubProjectProperty() throws IOException {
+        WorkflowJob job = createExampleJob();
+        String url = "https://github.com/jenkinsci/github-plugin/";
```

Comment:
Sorry to ask, but what would be the benefit of that? This value is not used in any other test case, so making it a property on the class does not make me feel like it has any obvious benefits.

Again, I'm no Java Expert, so if there's benefits to this, feel free to let me know. Learning new things is always good. :smile: